macros/commands/dump_table.sql

```diff
@@ -1,14 +1,24 @@
-{% macro dump_table(model_unique_id, output_path, since=none, days_back=7) %}
+{% macro dump_table(model_unique_id, output_path, exclude_deprecated_columns=true, since=none, days_back=7) %}
     {% set node = graph.nodes[model_unique_id] %}
-    {% set relation_exists = adapter.get_relation(database=node.database, schema=node.schema, identifier=node.alias) %}
-    {% if not relation_exists %}
+    {% set relation = adapter.get_relation(database=node.database, schema=node.schema, identifier=node.alias) %}
+    {% if relation is none %}
         {% do print("Relation '{}' does not exist.".format(node.relation_name)) %}
         {% do return([]) %}
     {% endif %}
 
+    {% set column_names = adapter.get_columns_in_relation(relation) | map(attribute="name") | list %}
+    {% if exclude_deprecated_columns %}
+        {% set deprecated_column_names = node.columns.values()| selectattr("deprecated") | map(attribute="name") | list %}
```

Comment:
Did you test this on anything? What happens if `node.columns.values()` doesn't have an attribute `"deprecated"` (because it's empty?) 
Or alternatively - can we be sure we won't encounter this case?

Reply:
Yes, tested.
We have only one column that does have it.